en/full-node.md

```diff
@@ -1287,6 +1287,26 @@ If you have any questions about configuring Bitcoin Core, please stop by
 one of our [forums](/en/bitcoin-core/help#forums) or [live
 chatrooms](/en/bitcoin-core/help#live).
 
+### Reduce Storage
+
+It is possible to configure your node to to run in pruned mode in order to
+reduce storage requirements. This can reduce the disk usage from over 100GB to
+around 2GB.
+
+Running a node in pruned mode is incompatible with `-txindex`
+and `-rescan` and disables the RPCs `importwallet`, `importaddress`,
+and `importprivkey`.
+
+To enable block pruning set `prune=<N>` on the command line or in `bitcoin.conf`,
+where `N` is the number of MiB to allot for raw block & undo data.
```

Comment:
Nitpicks: I think either the `<>` should be removed from the `prune=<N>` or they should be added to the `N` so that it's clear that they shouldn't be part of the actual entry.  Also I think `&` should be `and` in regular paragraph text.